tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -169,3 +169,180 @@ test.describe('Choose Template Type Page', () => {
     await expect(chooseTemplatePage.goBackLink).toBeHidden();
   });
 });
+
+test.describe('NHS App Message Template tests -in progress', () => {
+  const sessionStorageHelper = new SessionStorageHelper([
+    emptySessionData,
+    emptySessionDataForRadioSelect,
+    nhsAppRadioSelectedSessionData,
+  ]);
+
+  test.beforeAll(async () => {
+    await sessionStorageHelper.seedSessionData();
+  });
+
+  test.afterAll(async () => {
+    await sessionStorageHelper.deleteSessionData();
+  });
+  test('Validate error messages on the create NHS App message template page', async ({
+    page,
+    baseURL,
+  }) => {
+    const chooseTemplatePage = new TemplateMgmtChoosePage(page);
+
+    await chooseTemplatePage.navigateToChooseTemplatePage(emptySessionData.id);
+
+    await expect(page).toHaveURL(
+      `${baseURL}/templates/choose-a-template-type/${emptySessionData.id}`
+    );
+    expect(await chooseTemplatePage.fieldsetHeading.textContent()).toBe(
+      'Choose a template type to create'
+    );
+    await TemplateMgmtChoosePage.checkRadioButton(
+      page.locator('[id="templateType-NHS_APP"]')
+    );
+    await chooseTemplatePage.clickContinueButton();
+    await page.waitForTimeout(3000);
```

Comment:
we should avoid having fixed sleeps in the tests as it makes the tests take longer than they need to - if you need to wait, wait for a specific event or element on the page rather than a number of seconds

Reply:
removed waits.